### PR TITLE
Edited the remarks for RDFstar-demo.md 

### PR DESCRIPTION
**Pull request by RicardoUsbeck**
* Changed to 'RDF-star' from 'RDF*'.  Idem for 'SPARQL-star' vs 'SPARQL*'

* Changed "complex relationships" from "complex concepts"

* Renamed: man and :woman to :alice and :bob

* In 'N-ary relations' example: used lowercase for the instance :Marriage1

* Changed the 'Named Graphs' example to TriG

* Changed 'Expressing data certainty' as suggested to a certainty of 0.4 so that the triple is not asserted

* Did not change the following "'Expressing data value qualifiers': I am not a big fan of using RDF-star for expressing units. For this kind of use-case, I much prefer using a dedicated datatype (e.g. cdt:ucum), an intermediary node ([ rdf:value 32.1; :unit "cm" ]) or a dedicated property (:heightInCm 32.1)." @kmitd

* Added a longer, educational section about modelling for 'Expressing data value qualifiers' and 'Expressing data provenance'

* Altered and explained the SPARQL-bind example

* 'Coverting' → 'Converting'

### CODE REVIEW

File: RDFstar-demo.md [hunk 1]
```diff
@@ -1,66 +1,76 @@
...
 
-#### 3. Named Graphs
+#### 3. Named Graphs (TriG notation)
 
 ```
-:man :hasSpouse :woman :statementId#1 .
-:statementId#1 :startDate "2020-02-11"^^xsd:date :metadata .
+@prefix : <http://www.example.org/> .
+:statementId#1 { :bob :hasSpouse :alice }
+{ :statementId#1 :startDate "2020-02-11"^^xsd:date :metadata . }
```
Comment (pchampin) [inline RDFstar-demo.md]: That is still invalid TriG.
```suggestion
:metadata { :statementId#1 :startDate "2020-02-11"^^xsd:date }
```

**Review by pchampin — APPROVED** [review]

File: RDFstar-demo.md (hunk 1 above)
Comment (RicardoUsbeck) [reply RDFstar-demo.md]: Ah right. I wanted to make it the default graph not yet another graph with a name but forgot to delete ":metadata". Maybe remove ":metadata"?

Comment (pchampin) [reply RDFstar-demo.md]: yes, I am fine with removing :metadata as well... in which case the brackets are not even required.
```suggestion
:statementId#1 :startDate "2020-02-11"^^xsd:date .
```

**Comment by RicardoUsbeck** [comment]
Any updates @LongquanJiang ?

**Comment by LongquanJiang** [comment]
No

**Pull request merged**